sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4

```diff
@@ -899,7 +910,9 @@ unitToUnitInterval
     ;
 
 intervalValue
-    : (PLUS | MINUS)? (INTEGER_VALUE | DECIMAL_VALUE | STRING)
```

Comment:
nit: can we simply replace `STRING` with `stringLit` here?

Reply:
I am having a follow-up for this very soon.